src/ot_orchestration/operators/batch/vep.py

```diff
@@ -205,134 +248,3 @@ def _vep_command(self) -> list[str]:
                 --plugin AlphaMissense,file={self.pm.cache_dir}/AlphaMissense_hg38.tsv.gz,transcript_match=1 \
                 --plugin CADD,snv={self.pm.cache_dir}/CADD_GRCh38_whole_genome_SNVs.tsv.gz",
         ]
-
-
-class ConvertVariantsToVcfOperator(BaseOperator):
```

Comment:
As I understand,this is no longer needed, because this functionality is done by a separate step, right?

Reply:
This is no longer needed, as it is a part of the `variant_to_vcf` step -> https://github.com/opentargets/gentropy/blob/4d8e7c42e8e94473495d12d50efb7d7942afbe73/src/gentropy/variant_index.py#L114